src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -1043,6 +1043,37 @@ export class SearchView extends ViewletPanel {
 		this.onQueryChanged(true);
 	}
 
+	searchReplace(query?: string, replace?: string, triggerSearch?: boolean, caseSensitive?: boolean, wholeWords?: boolean, regex?: boolean, filesToInclude?: string, filesToExclude?: string): void {
```

Comment:
This is better but I'd like you to pass the full args object into here instead of having a bunch of arguments

Reply:
A name like `setSearchParameters` would be better

Reply:
Updated with feedback fixed